Shader fails to compile with commit 149bcba

[Commit 149bcba](https://github.com/servo/webrender/commit/149bcba51e345f56e444a0552cd39204ebbb6b90) broke webrender for me, because of shader compilation.

Error:
> Failed to compile shader: "..\\webrender\\webrender\\res\\ps_image_clip.fs.glsl"
> 0(15) : error C1008: undefined variable "pos"
> 0(27) : error C1008: undefined variable "relative_pos_in_rect"

[Commit e9ad82e](https://github.com/servo/webrender/commit/e9ad82efbac1bde805af45e9f2bff4c2792695f4) works, [Commit 149bcba](https://github.com/servo/webrender/commit/149bcba51e345f56e444a0552cd39204ebbb6b90) not, so it is safe to say that it is due to these changes.

OS: Windows 10
Driver: 372.54

--- a/webrender/res/ps_image_clip.fs.glsl
+++ b/webrender/res/ps_image_clip.fs.glsl
@@ -11,8 +11,8 @@ void main(void) {
 
     // We clamp the texture coordinate calculation here to the local rectangle boundaries,
     // which makes the edge of the texture stretch instead of repeat.
-    vec2 pos_for_texture =
-         clamp(pos, vLocalRect.xy, vLocalRect.xy + vLocalRect.zw) - vLocalRect.xy;
+    vec2 relative_pos_in_rect =
+         clamp(local_pos, vLocalRect.xy, vLocalRect.xy + vLocalRect.zw) - vLocalRect.xy;
 #else
     float alpha = 1.f;
     vec2 local_pos = vLocalPos;

